server.go

```diff
@@ -88,27 +103,34 @@ func (s *StubServer) HandleRequest(w http.ResponseWriter, r *http.Request) {
 
 	auth := r.Header.Get("Authorization")
 	if !validateAuth(auth) {
-		writeResponse(w, r, start, http.StatusUnauthorized,
-			fmt.Sprintf(invalidAuthorization, auth))
+		message := fmt.Sprintf(invalidAuthorization, auth)
+		stripeError := createStripeError(typeInvalidRequestError, message)
+		writeResponse(w, r, start, http.StatusUnauthorized, stripeError)
 		return
 	}
 
+	// Every response needs a Request-Id header except the invalid authorization
+	w.Header().Set("Request-Id", "req_123")
+
 	route := s.routeRequest(r)
 	if route == nil {
-		writeResponse(w, r, start, http.StatusNotFound, nil)
+		message := fmt.Sprintf(invalidRoute, r.Method, r.URL.Path)
+		stripeError := createStripeError(typeInvalidRequestError, message)
+		writeResponse(w, r, start, http.StatusNotFound, stripeError)
 		return
 	}
 
 	response, ok := route.operation.Responses["200"]
 	if !ok {
-		fmt.Printf("Couldn't find 200 response in spec\n")
```

Comment:
Shouldn't we keep this `fmt.Printf()` call?

Reply:
we should sorry I missed this one when reviewing my own diff, fixed.